gatsby/src/components/link/link.tsx

```diff
@@ -0,0 +1,33 @@
+import React, { useMemo } from 'react';
+import classnames from 'classnames';
+import { Link as GatsbyLink } from 'gatsby';
```

Comment:
poprosim
```suggestion
import {Link} from 'gatsby-plugin-react-i18next';
```
https://www.gatsbyjs.com/plugins/gatsby-plugin-react-i18next/#change-your-components